Detectors/Upgrades/ALICE3/Passive/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright CERN and copyright holders of ALICE O2. This software is distributed
+# under the terms of the GNU General Public License v3 (GPL Version 3), copied
+# verbatim in the file "COPYING".
+#
+# See http://alice-o2.web.cern.ch/license for full licensing information.
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+o2_add_library(A3DetectorsPassive
```

Comment:
Are we sure we want to call the targets `A3...` and not `ALICE3`? I would prefer to make sure that people know where this belongs.

Reply:
To me it makes no difference, was trying to shorten a bit the names.